modular_skyrat/modules/corporate_diplomat/code/subsystem.dm

```diff
@@ -0,0 +1,28 @@
+/datum/controller/subsystem/job
+	/// The role type that was chosen for corporate diplomat
+	var/datum/corporate_diplomat_role/corporate_diplomat_type
+
+/datum/job/proc/job_setup()
+	return
+
+/datum/job/corporate_diplomat/job_setup()
```

Comment:
Couldn't this just be done in `/datum/job/corporate_diplomat/New()` instead of adding a new proc?

Reply:
Guh. Probably, i'll get to this

Reply:
Done